doc/api/deprecations.md

```diff
@@ -2519,18 +2519,22 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`][] and
 [`request.socket`][] over [`request.connection`][].
 
-<a id="DEP0134"></a>
-### DEP0134: process._tickCallback
+<a id="DEP0XXX"></a>
+### DEP0XXX: process._tickCallback
 <!-- YAML
 changes:
   - version: REPLACEME
     pr-url: https://github.com/nodejs/node/pull/29781
     description: Documentation-only deprecation.
+  - version: REPLACEME
```

Comment:
I think new entries go at the top of the changes list.

Reply:
you are correct, updating